pilot/pkg/proxy/envoy/readiness.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 Istio Authors
```

Comment:
This will be linked in agent - we don't want to pull all ads stuff and deps and create confusion. Agent-linked code needs to be separate package ( we also need to untangle it from the other packages ).

Reply:
done.